core/vm/evm.go

```diff
@@ -200,10 +200,9 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 	debug := evm.Config.Tracer != nil
 
 	if !evm.StateDB.Exist(addr) {
-		if !isPrecompile && evm.chainRules.IsEIP4762 {
-			// add proof of absence to witness
+		if !isPrecompile && evm.chainRules.IsEIP4762 && value.Sign() != 0 {
 			gc := gasConsumer{availableGas: gas}
-			ok := evm.Accesses.TouchFullAccount(addr.Bytes(), false, gc.consumeGas)
+			ok := evm.Accesses.TouchCodeHash(addr.Bytes(), true, gc.consumeGas)
```

Comment:
In a `CALL` targeting an account that doesn't exist, we were doing this **read-event** on `TouchFullAccount`. This looks like it's missing charging the write we'll do on the **created** account `CODE_HASH=emptyCodeHash`.

But my code change is apparently a bit more weird, so let me explain.

There're two ways in which we can create an account via a call:
1. We're processing a value-bearing tx targeting an account that doesn't exist. In this case, [we already added the target ("for free") in the witness](https://github.com/gballet/go-ethereum/blob/jsign-hz/core/state_transition.go#L399). So this line will be a "noop" reg adding things to the witness and not chargin any extra gas. 
2. We're coming from a `CALL` instruction. In this case, we have a value-bearing `CALL` which means the `gasCall` already write-touched the `BASIC_DATA`. So, at this point _the only missing_ thing seems to be write-touching the `CODE_HASH`, thus why I removed the `FullAccount` to `CodeHash` in the touching. 

Note also I added `value.Sign() != 0` in the `if` since we only want to do this write-event on value-bearing contexts. If this "call" isn't value bearing, I don't see why we ned to do any new touching if `!Exist(addr)` since the `gasCall` already added everything needed in the witness to check for account existance check in stateless clients (if we agree stateless client doesn't need checking for codeHash for that). 

This is also why the deleted line L204 I believe was incorrect -- nothing in this place of hte code is about adding "proof of absence" .. this is indirectly done in the `gasCall`.